README.md

```diff
@@ -1,6 +1,6 @@
 # Pipeline HeavyJob Plugin
 
-This plugin allows defining job weight for pipeline projects as
+This plugin allows defining job weights for pipeline projects like
```

Comment:
Another option is "just as". But I thought that there are differences between plugins, so it's not a 100% match.